02_configure_host.sh

```diff
@@ -82,14 +99,27 @@ if [ "$INT_IF" ]; then
   sudo ip link set "$INT_IF" master baremetal 
 fi
 
-# Switch NetworkManager to internal DNS
-sudo mkdir -p /etc/NetworkManager/conf.d/
-sudo crudini --set /etc/NetworkManager/conf.d/dnsmasq.conf main dns dnsmasq
-if [ "$ADDN_DNS" ] ; then
-  echo "server=$ADDN_DNS" | sudo tee /etc/NetworkManager/dnsmasq.d/upstream.conf
-fi
-if systemctl is-active --quiet NetworkManager; then
-  sudo systemctl reload NetworkManager
-else
-  sudo systemctl restart NetworkManager
+if [ "$OOOQ_SETUP" == 'n' ]; then
+    cat > provisioning.xml << EOF
+    <network >
+      <name>provisioning</name>
+      <bridge name='provisioning' stp='on' delay='0'/>
+    </network>
+EOF
+    MAC=$(/usr/sbin/ip l show dev eth0 | awk '/link/ {print $2}')
+    cat > baremetal.xml << EOF
+    <network >
+      <name>baremetal</name>
+      <bridge name='baremetal' stp='on' delay='0'/>
+      <mac address="$MAC"/>
+    </network>
+EOF
+    sudo virsh net-define provisioning.xml
+    sudo virsh net-define baremetal.xml
+    sudo virsh net-start provisioning
+    sudo virsh net-start baremetal
+    echo -e "DEVICE=baremetal\nTYPE=Bridge\nHWADDR=$MAC\nONBOOT=yes\nNM_CONTROLLED=no\nBOOTPROTO=dhcp" | sudo dd of=/etc/sysconfig/network-scripts/ifcfg-baremetal
+    echo -e "DEVICE=eth0\nTYPE=Ethernet\nONBOOT=yes\nNM_CONTROLLED=no\nBOOTPROTO=none\nBRIDGE=baremetal" | sudo dd of=/etc/sysconfig/network-scripts/ifcfg-eth0
+    echo -e "DEVICE=eth1\nTYPE=Ethernet\nONBOOT=yes\nNM_CONTROLLED=no\nBOOTPROTO=none\nBRIDGE=provisioning" | sudo dd of=/etc/sysconfig/network-scripts/ifcfg-eth1
+    sudo systemctl restart network
```

Comment:
We added NODES_PLATFORM to allow skipping the creation of the VMs, I wonder if you set that (to anything other than the default "libvirt", can we get the desired outcome, e.g quickstart will only configure the networks and skip the VM creation?

It seems kind of weird to skip VM setup on the host but still configure the networks etc though.
It seems

Reply:
Great, I wasn't aware of the NODES_PLATFORM. I'm going to explore using it.